inspect.yaml

```diff
@@ -51,6 +51,42 @@ paths:
               schema:
                 $ref: "#/components/schemas/Error"
 
+  /:
+    post:
+      operationId: inspect_post
+      summary: Inspect DApp state via POST
+      description: |
+        This method sends an inspect-state request to the DApp backend passing the payload string available in the request body.
+        The payload string should be URL-encoded; the inspect server will decode the string to UTF-8.
```

Comment:
The payload should be a binary, not URL-encoded.

Reply:
Done at [82a6bc8](https://github.com/cartesi/openapi-interfaces/pull/2/commits/82a6bc8fac6c1ccd08c91e43e382d630a1dc5627).